bin/node/runtime/Cargo.toml

```diff
@@ -151,6 +152,7 @@ std = [
 	"pallet-society/std",
 	"pallet-recovery/std",
 	"pallet-vesting/std",
+	"frame-try-runtime/std",
```

Comment:
Hmmmm. Enabling features for optional dependencies is not supported :thinking: 

https://github.com/rust-lang/cargo/issues/8832

Reply:
Yeah, this was not needed after all. Thanks for the ref.